### PR TITLE
Added is_extension attribute to MAVField

### PR DESCRIPTION
**Pull request by python36**

### CODE REVIEW

**Comment by python36** [comment]
@tridge Could you please have a review?

File: generator/mavparse.py [hunk 1]
```diff
@@ -248,16 +249,18 @@ def start_element(name, attrs):
...
-                print_format = attrs.get('print_format', None)
-                enum = attrs.get('enum', '')
-                display = attrs.get('display', '')
-                units = attrs.get('units', '')
-                if units:
-                    units = '[' + units + ']'
-                instance = attrs.get('instance', False)
-                new_field = MAVField(attrs['name'], attrs['type'], print_format, self, enum=enum, display=display, units=units, instance=instance)
                 if self.message[-1].extensions_start is None or self.allow_extensions:
+                    check_attrs(attrs, ['name', 'type'], 'field')
```
Comment (peterbarker) [inline generator/mavparse.py:253]: This doesn't look right; did you mean to move all o fths code to within the "am I doing extensions now?" block?

**Comment by python36** [comment]
I simplified the commit. Please review.

**Comment by python36** [comment]
@peterbarker I simplified the commit. Please review.

**Pull request closed without merging**